ArduCopter/ReleaseNotes.txt

```diff
@@ -989,7 +989,7 @@ Changes from 4.3.6
     e) Memory corruption bug in the STM32H757 (very rare)
     f) RC input on IOMCU bug fix (RC might not be regained if lost)
 ------------------------------------------------------------------
-Copter 4.3.6 05-Apr-2023 / 4.3.6-beta1, 4.3.6--beta2 27-Mar-2023
```

Comment:
Hi @muramura,

Thanks for this, I think 4.3.6-beta2 was a real release but it only lasted for a few hours.  Could you change "4.3.6--beta1"
 to "4.3.6-beta2"?  After that I think we can merge this.

Reply:
So beta2 has been around for a few hours, hasn't it? I have fixed it.

Reply:
Hi @muramura,

Sorry to be nitpicky but -beta2 was released at the same time as beta1.  The changes made above make it appear like it was released at the same time as the stable.  I think the line should look like this:

`Copter 4.3.6 05-Apr-2023 / 4.3.6-beta1, 4.3.6-beta1 27-Mar-2023`

Reply:
I agreed with the comment. I have made the changes.

Reply:
Sorry it should have been, 

Copter 4.3.6 05-Apr-2023 / 4.3.6-beta1, 4.3.6-beta2 27-Mar-2023